locale/de-DE.json

```diff
@@ -16,9 +16,9 @@
   "filter": {
     "sort": {
       "default": "Sortieren",
-      "most_popular": "Beliebteste",
-      "price_desc": "Preis: höchster zuerst",
-      "price_asc": "Preis: niedrigster zuerst"
+      "most_popular": "Beste Ergebnisse",
+      "price_desc": "Preis absteigend",
```

Comment:
I think "absteigend" & "aufsteigend" is a little bit more complex to understand than "Preis: höchster zuerst". Normally it takes a few seconds of the ppl to understand what it means. So i would prefer either to leave it as is, or change it to sth like "Teuerste zuerst" "Günstigste zuerst" (but not sure if "Teuer" is a good word to use, since its a bit negative).

Reply:
I changed it now to "Höchster Preis zuerst" and "Niedrigster Preis zuerst".